_includes/sql-guide/table-create-eg-stringset-datatype.md

```diff
@@ -0,0 +1,7 @@
+```sql
```

Comment:
created for new concept example, plus ended up as example table for insert into stringset which didn't exist